### ISSUE TITLE
Docker build: no such package '@local_cuda//': The repository '@local_cuda' could not be resolved and referenced by '@cub_archive//:cub'

### ISSUE THREAD

**Issue by jennyfothergill** [tensorflow/tensorflow#48380]
<em>Please make sure that this is a build/installation issue. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:build_template</em>

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 18.04.5 LTS
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: no
- TensorFlow installed from (source or binary): source, branch r2.5
- TensorFlow version: r2.5
- Python version: 3.7
- Installed using virtualenv? pip? conda?: conda
- Bazel version (if compiling from source): 3.7.2
- GCC/Compiler version (if compiling from source): gcc 7.5.0
- CUDA/cuDNN version: 10.2/8.1.1
- GPU model and memory: n/a, but I want to use on P100 and V100 nodes

**Describe the problem**
I'm trying to compile tensorflow from source in a docker container, but I get the following error:
```
no such package '@local_cuda//': The repository '@local_cuda' could not be resolved and referenced by '@cub_archive//:cub'
```
See attached dockerfile
[dockerfile.txt](https://github.com/tensorflow/tensorflow/files/6274261/dockerfile.txt)

If line 101 in the above dockerfile is changed from `yes "" | ./configure && \` to ` ./configure && \`, the same issue persists.

**Any other info / logs**
See attached traceback
[traceback.txt](https://github.com/tensorflow/tensorflow/files/6274260/traceback.txt)

Thank you so much for your help with this issue!

**Comment by freedomtan** [comment]
@jennyfothergill  please check https://github.com/tensorflow/tensorflow/pull/48310

**Comment by npanpaliya** [comment]
@freedomtan and @jennyfothergill  - Please check new PR for the same issue https://github.com/tensorflow/tensorflow/pull/48393. I closed the earlier one due to merge issues.

**Comment by jennyfothergill** [comment]
This issue is still not resolved by #48393 ...

**Comment by jennyfothergill** [comment]
Hi, sorry for the delay I am not as familiar with this issue anymore. @RainierBarrett has successfully compiled tensorflow in [this container](https://github.com/cmelab/containers/blob/main/containers/dockerfile_gputensorflow). He did have to [manually link certain cuda libs](https://github.com/cmelab/containers/blob/4fa231eb8f6543ea46723108939b5fecb8682ef3/containers/dockerfile_gputensorflow#L82-L88), but otherwise--yes I think this issue is resolved. 

Thank you for your help!
